src/command.rs

```diff
@@ -188,7 +200,11 @@ pub async fn publish(
 
     let artifactory = Artifactory::new(registry, &credentials)?;
 
-    let package = PackageStore::release().await?;
+    let manifest = Manifest::read().await?;
+    let package = PackageStore::current()
+        .into_diagnostic()?
+        .release(manifest)
```

Comment:
Can we split the releasing out? As we figured out yesterday its not really related to the package store

Reply:
Maybe we can put it into `Package` for now` Like `Package::release` or alike

Reply:
Sure, but that is something that should go into a separate PR. I don't want this one to be too big.